Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -29,6 +29,8 @@ class MessageDTO: NSManagedObject {
 
     @NSManaged var latestReactions: [String]
     @NSManaged var ownReactions: [String]
+    
+    @NSManaged var translations: [String: String]?
```

Comment:
Can it be non-optional and empty in case of no translations available?

Reply:
I choose this way so DB schema correctly reflects backend. IMHO it'd be easier to debug this way if an issue occurred